### DISCUSSION TITLE
OpenMCT integration with  NASA software and systems used in mission control

### DISCUSSION THREAD

**Discussion by ShivaBhattacharjee** [nasa/openmct#7402] [Community Q&A] [answered]
I'm curious about how OpenMCT and NASA's existing software team up during mission control. like how do they talk to each other and what steps are taken to ensure they get along?

**Comment by ozyx** [comment, marked as answer]
Hi Shiva,

Please check out [openmct-quickstart](https://github.com/scottbell/openmct-quickstart) for an example of a full-stack usage of Open MCT with YAMCS mission control software.

Cheers!
